server.py

```diff
@@ -18,4 +34,12 @@
 if os.name == 'nt':
     subprocess.run(['waitress-serve', '--port=3001', 'bot_python_sdk.api:api'])
 else:
-    subprocess.run(['gunicorn', '-b', '127.0.0.1:3001', 'bot_python_sdk.api:api'])
+    cmd = subprocess.Popen(['hostname', '-I'], stdout=subprocess.PIPE)
+    IPAddr = cmd.communicate()[0].decode('ascii').split(' ')[0]
```

Comment:
can you rename to `ipAddress` or just `ip`

lets not abbreviate of we don't have to.

Reply:
Right, done.